tests/Core/Tokenizers/Tokenizer/RecurseScopeMapDefaultKeywordConditionsTest.php

```diff
@@ -283,47 +283,47 @@ public static function dataSwitchDefault()
     {
         return [
             'simple_switch_default'                                     => [
-                'testMarker'   => '/* testSimpleSwitchDefault */',
-                'openerOffset' => 1,
-                'closerOffset' => 4,
+                'testMarker'       => '/* testSimpleSwitchDefault */',
+                'testOpenerMarker' => '/* testSimpleSwitchDefault */',
+                'testCloserMarker' => '/* testSimpleSwitchDefault */',
```

Comment:
The keys in the sub-arrays need to match the parameter names.
Otherwise, in a limited set of currently supported PHPUnit versions + on PHPUnit 10+, this will cause problems with (function calls with) named parameters.

Reply:
Thanks for the review, @jrfnl, I pushed a new commit fixing the keys in the sub-arrays in the data provider to match the parameter names in the test method.